src/loops/nested_loops/hard/q7/BoxMeNow.java

```diff
@@ -0,0 +1,39 @@
+package loops.nested_loops.hard.q7;
+
+/*
+Write a program called BoxMeNow to ask the use for n, and print the following pattern using nested loops based on their
+input. If the input is less than zero print “Invalid input!”
+Sample Output for n =  4:
+Enter number of lines:
```

Comment:
This should probably be the side length of the box, as we don't want anyone to get confused and only print four at the top.

Reply:
I'm confused. The number is the number on the sides of the box and the sample output shows it.